### PR TITLE
Update Chinese Translation -- Issue #57

### PR DESCRIPTION
**Pull request by davidrhodus**

### CODE REVIEW

**Comment by davidrhodus** [comment]
@NukeManDan @shawntabrizi

**Comment by NukeManDan** [comment]
@jimmychu0807 would you mind a quick review here? 😀

**Comment by NukeManDan** [comment]
We will also need to bring back the tab here, uncommenting the item

**Review by NukeManDan — APPROVED** [review]
I don't read fluently. But I trust this is good to go. I would still like a quick native speaker to do a quick review 🙏🏼

**Comment by jimmychu0807** [comment]
@davidrhodus thank you for the PR on Chinese translation. I will review it in about coming two days.

I saw from your github repo that it seems you are picking up on Substrate? How's your learning experience so far? Mind share more with me? Feel free to contact me in [Element] at @jimmychu0807:matrix.parity.io , and do join our [Substrate Technical channel](https://matrix.to/#/!HzySYSaIhtyWrwiwEV:matrix.org?via=matrix.parity.io&via=matrix.org&via=corepaper.org).

**Comment by jimmychu0807** [comment]
@davidrhodus Hi David, thank you for your effort. I took a look in your PR. It seems to me as unfinished work. Is this something still work in progress?

Screenshot 2021-05-22 at 11 36 45

For example, on the above screenshot.

1. The side menu doesn't match
2. The content structure in the Chinese doesn't quite match the corresponding page in english. Which commit of the english content do you base your translaton from?

**Review by jimmychu0807 — CHANGES_REQUESTED** [review]
See comment above

**Comment by davidrhodus** [comment]
> @davidrhodus Hi David, thank you for your effort. I took a look in your PR. It seems to me as unfinished work. Is this something still work in progress?
> 
> Screenshot 2021-05-22 at 11 36 45
> 
> For example, on the above screenshot.
> 
> 1. The side menu doesn't match
> 2. The content structure in the Chinese doesn't match the corresponding page in english. Which commit of the english content do you base your translaton from?

Thanks for the feedback, this is very helpful. I am new to markdown and missed comparing a couple of the files.

I will update the PR and include the following:

(1)
- Will update the _sidebar.md file 
- Will the README.md ( I mistakenly did not look at this file. )
- Will translate SettingUpTheBootnode.md 

(2)
I used the master branch for the english content. Please let me know if this is correct, or if I should use a specific branch or commit hash.

**Comment by jimmychu0807** [comment]
> I used the master branch for the english content. Please let me know if this is correct, or if I should use a specific branch or commit hash.

Yes the english content in master branch is correct. You can always run the site locally with `yarn install && yarn start` and compared the chinese against the english version see if they match.

**Comment by NukeManDan** [comment]
@davidrhodus I expect #68 to be very close to finished, and it includes a few new sections. Could this be used to generate the remaining translations once it's merged? 🙏🏼

**Comment by davidrhodus** [comment]
> See comment above

Hi @jimmychu0807 can you review my latest addition to the pull request? I think everything is covered.

**Comment by davidrhodus** [comment]
> @davidrhodus I expect #68 to be very close to finished, and it includes a few new sections. Could this be used to generate the remaining translations once it's merged? 🙏🏼

Thanks @NukeManDan for the heads up, I will start the translation on this next.

**Comment by jimmychu0807** [comment]
@davidrhodus Could you merge in your PR this branch https://github.com/substrate-developer-hub/cumulus-workshop/tree/zh-57? This is the edit I am working on based on your PR. I have finished review section one. Will do the rest in this or next week. But meanwhile, you could also have a review to make sure all sections are translated based on the `master` branch. I found some section missing.

**Comment by davidrhodus** [comment]
> See comment above

Updated

**Comment by NukeManDan** [comment]
@jimmychu0807 when you have a moment to review 🙏🏼

**Review by NukeManDan — APPROVED** [review]
I need to have a native speaker take a look 🙇🏼 , but assuming that passes I approve 😁

File: zh-cn/3-parachains/4-more-nodes.md [hunk 1]
```diff
@@ -1,51 +1,48 @@
 # 链接添加的平行链节点
 
-就像我们之前展示的，一条平行链 _只能_ 使用一个校对节点。但是这种配置并不是非常去中心化。一个恶意的对方只要关闭单个节点就可让平行链失控。
 
+正如我们已经展示的那样，平行链 _can_ 仅使用一个整理器。但是那个配置是不是很分散。攻击者只需要关闭一个节点即可阻止平行链。
```
Comment (jimmychu0807) [inline zh-cn/3-parachains/4-more-nodes.md:4]: 正如我们已经展示的那样，平行链 _仅_ 使用一个整理器。但是這種配置不是很分散。攻击者只需要关闭一个节点即可阻止平行链。

File: zh-cn/6-register/1-register.md [hunk 2]
```diff
@@ -6,177 +6,139 @@ Rococo是基于Cumulus的平行链，是Parity的官方测试网。
...
---name <your-node-name> # set a unique validator node name here
-# --telemetry-url 'wss://telemetry.polkadot.io/submit/ 0' \ # *optional* telemetry 
-```
+Rococo 测试网络的本地货币的符号是 ROC。您将需要一些 ROC 才能
+注册您的洛可可平行链。
 
-You will need to generate session keys for each of your Rococo validator nodes. To generate keys for
-a node, use the `author.rotateKeys` RPC call. One way to invoke this RPC method is by using `curl`:
-您需要给每个Rococo验证节点产生密钥。使用`author.rotateKeys` RPC call来产生密钥。调用此RPC方法的一种方法是使用`curl`：
+要在洛可可 _before_ 上进行实验，您需要注册，并鼓励您的团队作为支持者
```
Comment (jimmychu0807) [inline zh-cn/6-register/1-register.md:23]: Rococo 這名字就不需要翻成 `洛可可` 了

File: zh-cn/6-register/1-register.md [hunk 3]
```diff
@@ -6,177 +6,139 @@ Rococo是基于Cumulus的平行链，是Parity的官方测试网。
...
-_Output Example_:
-```json
-{
-  "jsonrpc":"2.0",
-  "result":"0xcd1e2d16bba602f5247b7e220ca15fc6e5ace0181c51aab510f16aeb358510cdfc9b39c84874e3b4025d29c3a23b4624cc4ea22335f431e3510f088f0514a64864da893bda27d6cdfa797f1d8a2e8cc4b49d90517587bbd215705e34fa1bac790825912435517e8cc7b107b71f4744411abb4740087c95415e712314e6dc2b4d9aa7c891718f08903018f8de983dd1648c98c85f4f7ba41a495a0d656acf9f6108aa481e5374203fc8a035ea9d94a62c5821a575e4515f74a799e60878765d04023bf989b976b5e2da7c42d76a03e4fd2aa4ccbdecc8237cdc3a887b2a97619d96",
-  "id":1
-}
-```
+由于您将需要比水龙头实际允许注册的更多，请加入并询问
+[洛可可矩阵聊天室](https://matrix.to/#/#rococo:matrix.parity.io)，Parity团队之一
```
Comment (jimmychu0807) [inline zh-cn/6-register/1-register.md:28]: 前两个子也不用翻，直接用：Rococo matrix 聊天室

File: zh-cn/6-register/1-register.md [hunk 4]
```diff
@@ -6,177 +6,139 @@ Rococo是基于Cumulus的平行链，是Parity的官方测试网。
...
+- 租期长度：1 天 = 14400 块
+- 结束时间：60 分钟 = 600 块
+- 当前租赁期指数 = 当前区块编号 / 14400
 
->注意：从Para *到不同状态的所有转换将至少需要2个会话（入门，离线，升级，降级等）
+> 注意：Para* 到不同状态的所有转换至少需要 2 个会话（入职、
+> 下线、升级、降级等
 
-### 投标
+＃＃＃ 投标
```
Comment (jimmychu0807) [inline zh-cn/6-register/1-register.md:86]: 得用 半型的 `#` 号。這是 markdown 格式。

**Review by jimmychu0807 — APPROVED** [review]
I have some minor comments on the wording. But overall it is much better and more up to date than the text in the existing version. 

@davidrhodus thanks again.

**Comment by NukeManDan** [comment]
@davidrhodus - would you be willing to rebase on #77 and tweak the langue for the translation? It's very minor here 🙏🏼
Plus Jimmy's comments, then we are good to get this merged 👍🏼

**Comment by davidrhodus** [comment]
> @davidrhodus - would you be willing to rebase on #77 and tweak the langue for the translation? It's very minor here 🙏🏼
> Plus Jimmy's comments, then we are good to get this merged 👍🏼

Yes, I'll get this completed over the next day. Thanks for the ping and the new updates. 🙂

**Comment by NukeManDan** [comment]
#77 is in, this PR should be ready to go on top of that 🚀

**Comment by NukeManDan** [comment]
migrating to #79

**Pull request closed without merging**